recipe/meta.yaml

```diff
@@ -12,9 +12,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  skip: true  # [py>=32]
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  number: 1
+  script: "{{ PYTHON }} -m pip install . -vv"  # [py<32]
```

Comment:
I don’t think we can have a selector here. We would need to add a version constraint to `python` in `host` and `run`.

Reply:
https://github.com/conda-forge/enum34-feedstock/blob/master/recipe/meta.yaml hasn't broken things yet ;)